Gems/Atom/RPI/Code/Source/RPI.Public/RPISystem.cpp

```diff
@@ -249,23 +249,25 @@ namespace AZ
 
             AssetInitBus::Broadcast(&AssetInitBus::Events::PostLoadInit);
 
-            // Update tick time info
-            FillTickTimeInfo();
+            m_currentSimulationTime = GetCurrentTime();
 
             for (auto& scene : m_scenes)
             {
-                scene->Simulate(m_tickTime, m_simulationJobPolicy);
+                scene->Simulate(m_simulationJobPolicy, m_currentSimulationTime);
             }
         }
 
-        void RPISystem::FillTickTimeInfo()
+        float RPISystem::GetCurrentTime()
```

Comment:
Makes sense that this would be a lot more accurate. Wondering if it would be good to make the interface use doubles by default. Based upon this convo : https://amzn-aws.slack.com/archives/G01CU9R1GP3/p1632514360019100

Reply:
Linking @amzn-sean who is doing some work on core time systems.

Reply:
Ok, so if i understand this function correctly. You're looking for the time from start of the application of the current onTick?
If so. you'll be able to simplify this after i get my timer changes merged (i'll make a note to loop back here).
```
AZ::TimeMs lastGameTickTime = AZ::GetLastGameTickTime();
return AZ::TimeMsToSeconds(lastGameTickTime);
```
If you do update to doubles, there is a ```AZ::TimeMsToSecondsDouble``` version as well.

I'm going to be opening the PRs soon on that change, so for now feel free to merge this change for the interm.

Reply:
As discussed offline, the changes @amzn-sean is making probably will go into development soon but would require an exception to go into the next release at this stage. So let's move forward with these changes in the meantime. We can improve the code later to take advantage of the new timing functions.